### ISSUE TITLE
Implement pitch shift transform

### ISSUE THREAD

**Issue by iver56** [asteroid-team/torch-audiomentations#66]

**Comment by KentoNishi** [comment]
i was looking to see if PitchShift was implemented in torch-audiomentations, and it seems like it isn't. is this in the works? if not, i might go ahead and implement it myself and submit a pull request

**Comment by iver56** [comment]
Hi @KentoNishi, and thanks for your interest in torch-audiomentations!

I wish we had PitchShift. It's not in the works right now as far as I know. You're welcome to submit a pull request. How are you planning to do it? Implement the pitch shift algorithm yourself? Or use the one in Wavaugment, like Spijkervet does?

* https://twitter.com/Spijkervet/status/1292411014584180736
* https://github.com/Spijkervet/torchaudio-augmentations/blob/master/torchaudio_augmentations/augmentations/pitch_shift.py#L23

Anyway, I'm looking forward to your contribution

**Comment by KentoNishi** [comment]
copy pasting from the slack channel:

```
I've been working on it for a couple hours now, and it seems like it works. I've also made a separate library for pitch shifting in pytorch, so i'll try dropping that into torch-audiomentations soon :)
tldr is that i implemented the stretch-and-resample technique that librosa uses, just using (mostly) torch tensor operations
```

right now i'm working in my own fork: https://github.com/KentoNishi/torch-audiomentations
i also made `torch_pitch_shift` a separate library (not public yet, will do after i'm done cleaning it up): https://pypi.org/project/torch-pitch-shift

**Comment by KentoNishi** [comment]
@iver56 this is a bit of a noob question, but I'm stuck at creating a uniform distribution of integers instead of floats. do you know how to do that? XD

**Comment by KentoNishi** [comment]
ah i can just floor the float distribution, stinky brain

**Comment by iver56** [comment]
Closed by https://github.com/asteroid-team/torch-audiomentations/pull/99
